docs/config/lua/config/font_rules.md

```diff
@@ -45,48 +45,53 @@ return {
 }
 ```
 
-Here's an example from my configuration file;
+Here's an example from a configuration file;
 
 ```lua
 local wezterm = require 'wezterm'
 
 -- A helper function for my fallback fonts
 function font_with_fallback(name, params)
-  local names = { name, 'Noto Color Emoji', 'JetBrains Mono' }
+  local names = { name, 'Noto Color Emoji', 'Symbols Nerd Font Mono' }
```

Comment:
Isn't the symbols nerd font happen anyways?

Reply:
https://github.com/ryanoasis/nerd-fonts/tree/498ebf372bfc84731df6fe9411c909b4e97b1ec0/patched-fonts/NerdFontsSymbolsOnly/complete

That depends, for my setup I only have this, and only want to use Mono.

Reply:
It works for me, unless I have it Mono here, the icons become "dull".

Reply:
`Noto Color Emoji` , `JetBrains Mono` and `Symbols Nerd Font Mono` are always automatically appended to the list of fonts, whether you use `wezterm.font` or `wezterm.font_with_fallback`, so this example could be simplified by removing these.

This helper function is more useful when you have fonts that are not part of the automatic fallback that you wish to reference from multiple `font_rules`.